package.json

```diff
@@ -6,7 +6,8 @@
   "license": "Apache-2.0",
   "scripts": {
     "clean": "rimraf coverage build tmp dist",
-    "build": "ncc build src/main.ts -o dist --source-map --license LICENSE",
+    "prebuild": "yarn run clean",
+    "build": "ncc build src/main.ts -e typescript -o dist --source-map",
```

Comment:
`-e typescript` removes the typescript from `dist/`

Reply:
@stack72 When using `-e typescript`, the workflow fails.

```shell
Error: Cannot find module 'typescript'
Require stack:
- /home/runner/work/_actions/pulumi/actions/add-basics/dist/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.5034 (/home/runner/work/_actions/pulumi/actions/add-basics/webpack:/pulumi-github-action/external "typescript":1:1)
    at __nccwpck_require__ (/home/runner/work/_actions/pulumi/actions/add-basics/webpack:/pulumi-github-action/webpack/bootstrap:20:1)
    at Object.8338 (/home/runner/work/_actions/pulumi/actions/add-basics/webpack:/pulumi-github-action/node_modules/@pulumi/pulumi/runtime/closure/parseFunction.js:16:1)
    at __nccwpck_require__ (/home/runner/work/_actions/pulumi/actions/add-basics/webpack:/pulumi-github-action/webpack/bootstrap:20:1)
    at Object.1260 (/home/runner/work/_actions/pulumi/actions/add-basics/dist/index.js:35176:25)
    at __nccwpck_require__ (/home/runner/work/_actions/pulumi/actions/add-basics/webpack:/pulumi-github-action/webpack/bootstrap:20:1)
```

Reply:
https://github.com/vercel/ncc/issues/642

Reply:
So the typescript parameter is not required? If so this is a great news!